File: eco-monitoring-back/src/main/java/com/example/ecomonitoringback/model/Pollutant.java
package com.example.ecomonitoringback.model;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "pollutants")
public class Pollutant {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private Double minMass;
    private Double maxMass;
    private Double gdk;
    private Integer dangerClass;
}

File: angular/src/app/model/Pollutant.ts
export class Pollutant {
  id: number = -1;
  name: string = "";
  minMass: number = 0;
  maxMass: number = 0;
  gdk: number = 0;
  dangerClass: number = 0;
  rfc: number = 0;
  sf: number = 0;
  organs: string = "";
  taxRatesByStationarySources: number = 0; // Ставка податку, гривень за 1 тонну
  taxRatesForDischargesToWaterBodies: number = 0; // Ставка податку, гривень за 1 тонну
  taxRatesForWasteDisposal: number = 0; // Ставка податку, гривень за 1 тонну
  taxRatesForTheGenerationOfRadioactiveWaste: number = 0;
  taxRatesForTemporaryStorageOfRadioactiveWaste: number = 0;
}
